app/components/MobilePanelSizeControl.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, BoxProps, Hide } from "@nycplanning/streetscape";
+
+export interface MobilePanelSizeControlProps extends BoxProps {
+  isExpanded: boolean;
+  isExpandedToggle: () => void;
+}
+export function MobilePanelSizeControl({
```

Comment:
Nit: we might want a more descriptive name, just because I couldn't figure out what this actually was meant to be until I looked at the Box component closely. Maybe something like `ExpandBar` or `ResizeBar`.